index.html

```diff
@@ -71,7 +71,7 @@ <h2>Haiku</h2>
         </p>
     </footer>
     <div class="back-to-top">
-        <a href="" ud="top">◤</a>
+        <a href="#top">◤</a>

```

Comment:
You don't need an id here.  The href is what points to the id you set on the `<header>` above using the hashtag syntax (similar to the css selector for an id).